app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,14 @@
+<% provide(:title,t(".about")) %>
+
+<h1>About</h1>
+<p>
+  The <a href="https://www.railstutorial.org/"><em>Ruby on Rails
```

Comment:
define i18n cho các chuỗi hiển thị lên view, fix toàn bộ các file view đang chưa có
áp dụng kỹ thuật https://guides.rubyonrails.org/i18n.html#using-safe-html-translations để chèn code HTML vào chuỗi